code/modules/power/cell.dm

```diff
@@ -205,6 +205,12 @@
 /obj/item/cell/hyper/empty
 	charge = 0
 
+/obj/item/cell/hyper/adherent
+	name = "crystal-ceramic grid"
+	desc = "A dense, smooth blue polygon the size of a clenched fist. It's etched with symmetrical facets that are filled with a thick, gelatinous material."
```

Comment:
```suggestion
	desc = "A dense, smooth, blue polygon the size of a clenched fist. It's etched with symmetrical facets that are filled with a thick, gelatinous material."
```

Reply:
I believe this is still proper grammar, and I feel like it rolls off the tongue better. "a dense, smooth blue polygon" implies "it is a smooth blue polygon that is also dense" rather than "it is a blue polygon that is also dense and smooth". Weird wording semantics, but I prefer it as is.

Reply:
Commas are required between coordinate adjectives; the phrase "a light, plastic, orange pen" for example. Having three adjectives modifying one noun is admittedly awkward and prone to semantical issues. It's your choice and I doubt it will spill anyone's tea.